### DISCUSSION TITLE
Grug-far keymap constantly overrides my keymap

### DISCUSSION THREAD

**Discussion by banjo** [LazyVim/LazyVim#4571] [Q&A] [answered]
I have remapped grug-far in my keymap folder, but whenever I open the grug-far application in Nvim I get the default LazyVim grug-far keymap which overrides my current map. How can I disable the default keymap when importing grug-far?

I have added the `editor` plugin and also added this config for grug-far manually:

```
return {
  "MagicDuck/grug-far.nvim",
  opts = { headerMaxWidth = 80 },
  cmd = "GrugFar",
  keys = {},
}
```

In keymap.lua

```
-- grug-far
vim.keymap.set("n", "<leader>sR", function()
  local grug = require("grug-far")
  local ext = vim.bo.buftype == "" and vim.fn.expand("%:e")
  grug.grug_far({
    transient = true,
    prefills = {
      filesFilter = ext and ext ~= "" and "*." .. ext or nil,
    },
  })
end, {
  desc = "[s]earch [R]eplace (grug-far)",
})
```

Basically, grug-far overrides `<leader>sr`, which I already map in the telescope file:

```
      { "<leader>sr", "<cmd>Telescope resume<cr>", desc = "[s]earch [r]esume" },
```

**Comment by dpetka2001** [comment, marked as answer]
Just put in your `grug-far` plugin spec in `keys`
```lua
{ "<leader>sr", false },
{ "<leader>sR", your_function_here}
```
and in Telescope plugin spec do also the same.

**Reply by banjo** [reply]
That worked, thanks!
